app/src/main/java/com/example/androidenhancedvideoplayer/components/RecommendedVideoComponent.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.androidenhancedvideoplayer.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.example.androidenhancedvideoplayer.R
+
+@Composable
+fun RecommendedVideoComponent(
+    videoTitle: String = "Video Title",
+    thumbnailResId: Int = R.drawable.ic_launcher_background,
+    duration: String = "00:40"
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(16.dp)
+    ) {
+        Image(
+            painter = painterResource(thumbnailResId),
+            contentDescription = null,
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(180.dp)
+                .clip(shape = RoundedCornerShape(8.dp)),
+            contentScale = ContentScale.Crop
+        )
+
+        Spacer(modifier = Modifier.height(8.dp))
+
```

Comment:
We can add it as a padding also, but doing this if the component has some animation it would animate the empty space also, which may not be the intended action.